src/Microsoft.DotNet.Darc/src/DarcLib/Actions/Remote.cs

```diff
@@ -642,9 +644,9 @@ private IEnumerable<DependencyDetail> CalculateLeavesOfCoherencyTrees(IEnumerabl
             CheckForValidGitClient();
             GitFileContentContainer fileContainer =
                 await _fileManager.UpdateDependencyFiles(itemsToUpdate, repoUri, branch);
-            List<GitFile> filesToCommit = fileContainer.GetFilesToCommit();
+            List<GitFile> filesToCommit = new List<GitFile>();
```

Comment:
Is this going to skip over the version files if there is no arcade sdk item?

Reply:
Yep. Good catch. Meant for the files in the FileContainer to be added regardless. Fixed now